From: Rinigus <rinigus.git@gmail.com>
Date: Sun, 4 Aug 2019 20:30:41 +0300
Subject: use pyflakes3 in Debian

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index adc1920..de26eee 100644
--- a/Makefile
+++ b/Makefile
@@ -26,7 +26,7 @@ define install-translation =
 endef
 
 check:
-	pyflakes geocoders guides poor routers
+	pyflakes3 geocoders guides poor routers
 	find . -type f -name "*.json" -exec jsonlint -q {} \;
 
 clean:
